src/hydration.js

```diff
@@ -0,0 +1,36 @@
+class Hydration {
+  constructor(hydrationData, id) {
+    this.data = hydrationData;
+    this.id = id;
+    this.user = this.findUserHydroData();
+  };
+
+  findUserHydroData() {
+    return this.data.filter(user => user.userID === this.id)
+
+  }
+
+  calculateAvgOzsPerDay() {
+    let totalOz = this.user.reduce((total, day) => {
+        total += day.numOunces
+        return total
+      },0);
+    return Math.floor(totalOz/this.user.length)
+
+  };
+
+  calculateDailyOz(date) {
+    let selectedDay = this.user.find(day => day.date === date)
+    return selectedDay.numOunces;
+  };
+
+  calculateWeeklyOz(date) {
```

Comment:
This method was tricky to nail down.  We finally were able to get it to work by identifying the index position of the date that is passed in as a parameter.  Then we were able to create an object with the full week of entries by taking the date passed in and slicing the previous 6 days plus the date passed in as a parameter.